Data-not-files: IPFS 101, FAQ, & Hashing changes (#1022)
ipfs/ipfs-docs: docs/concepts/faq.md

For installing and initializing IPFS from the command line, check out the [command-line quick start](../how-to/command-line-quick-start.md) guide.

> ### Why doesn't my SHA hash match my CID?

Reviewer: ```suggestion
### My SHA hash is different to my CID

```

Try not to add symbols into titles. And add an empty line after a title and a paragraph.

Reviewer: > Sounds like our system has a hard time with symbols

Yeah, it comes down to markdown translating headers into HTML anchors, and throwing symbols into the mix can get varied results depending on which compiler is used. So the easiest option is to remove symbols altogether.

Saying that, you're right; since this is an FAQ we should probably leave them in here, and just deal with compiler issues down the road (if they happen at all).

> When you add a file to IPFS, IPFS splits it into smaller blocks. Each of these pieces is hashed individually, which then results in an overall different hash. Instead, IPFS uses Merkle DAGs, which are self-verifiable. See [Merkle Directed Acyclic Graphs (DAGs)](../concepts/merkle-dag.md).

Reviewer 2: The "instead" seems to be misplaced. I propose something like:

```suggestion
When you add a file to IPFS, IPFS splits it into smaller blocks. Each of these pieces is hashed individually, and build a [Merkle DAG](../concepts/merkle-dag.md), which then results in an overall different hash. 
```
